src/Microsoft.AspNetCore.Razor.Sdk/build/netstandard2.0/Microsoft.AspNetCore.Razor.Sdk.props

```diff
@@ -0,0 +1,8 @@
+<Project>
+  <Import Project="Sdk.CurrentVersion.props" />
+
+  <PropertyGroup>
+    <!-- Redirect the SDK to use the targets from the current package -->
+    <_MicrosoftAspNetCoreRazorSdkCurrentVersionPath>$(MSBuildFileDirectory)Sdk.CurrentVersion.targets</_MicrosoftAspNetCoreRazorSdkCurrentVersionPath>
```

Comment:
Think about just simplifying this name too, just like `RazorSdkCurrentVersionPath`